pr2eus_tutorials/launch/pr2_tabletop.launch

```diff
@@ -19,25 +33,49 @@
     <arg name="publish_tf" value="$(arg publish_objects_tf)" />
   </include>
 
-  <node name="pointcloud_screenpoint_nodelet" pkg="nodelet" type="nodelet"
-        args="load jsk_pcl/PointcloudScreenpoint $(arg manager)"
-        output="screen" clear_params="true" respawn="true"
-        machine="$(arg machine)">
-    <remap from="~points" to="/kinect_head/depth_registered/throttled/points" />
-    <remap from="~point" to="/kinect_head/rgb/throttled/image_rect_color/screenpoint" />
+  <group if="$(arg remote)">
```

Comment:
I think `<group if="$(arg remote)">` should include xxx_decompress nodes and depth_image_proc nodes.

Reply:
What a mess!!!  Fix with [7fbc86ea94fd3e760502e4ecee27c0339495c80f](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/7fbc86ea94fd3e760502e4ecee27c0339495c80f) and 
[7fda6c29b3b68c505ece11a15ebfa29fdb140c42](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/7fda6c29b3b68c505ece11a15ebfa29fdb140c42)